test/controllers/pruebas_controller_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'test_helper'
+
+class PruebasControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @prueba = pruebas(:one)
+  end
+
+  test "should get index" do
```

Comment:
I didn't find the resource, but I like to divide my test in 3 blocks, prepare, actions and assertions.
This can be rewritten as:
```
get pruebas_path

assert_response :success
```